src/python/pants/bin/local_pants_runner.py

```diff
@@ -157,6 +155,12 @@ def set_start_time(self, start_time):
     self._run_start_time = start_time
 
   def run(self):
+    # Apply exiter options.
+    # TODO: this is messy. Is it less surprising to have the Exiter modified during construction, or
+    # in the run() method we do here?
+    ExceptionSink.set_destination(self._global_options.pants_workdir)
+    self._exiter.should_print_backtrace = self._global_options.print_exception_stacktrace
```

Comment:
Question I do not know the answer to: should the `should_print_backtrace` property be on the `ExceptionSink` instead? It applies to the current `errstream`, rather than to the Exiter, per-se.

Reply:
I moved this around many times, including letting the traceback formatter be None to denote not printing tracebacks. On master, `should_print_backtrace` applies to the exception message printed to stderr on exit only, not to the log message, so this code conforms to the current status, which also seems to make most sense to me (I was going to delete it, but obviously this is required for the specific case of exiting with error). We take an Exiter as a param in order to avoid subsuming exiting as well as fatal logging functionality. The current `errstream` is an implementation detail, and btw, also not used yet, I need to make ExceptionSink call its Exiter with the `out` arg provided. I don't think `should_print_backtrace` *should* be a property of the `errstream` or the `ExceptionSink` as opposed to the Exiter unless there are more situations besides "we are printing an error message on an unexpected error, possibly to a terminal" that we would want `should_print_backtrace` to apply to (which I'm all ears for, but don't see right now).

Note to self to make ExceptionSink use the `out` arg when exiting.